apps/ejabberd/src/mod_pubsub.erl

```diff
@@ -235,6 +236,10 @@ stop(Host) ->
     ok = supervisor:terminate_child(ejabberd_sup, Proc),
     ok = supervisor:delete_child(ejabberd_sup, Proc).
 
+-spec default_host() -> string().
+default_host() ->
+    "pubsub.@HOST@".
```

Comment:
1) In most places it was still provided as string.
2) It's set with a string in config file.
3) It doesn't make a difference for `re:replace`